.travis.yml

```diff
@@ -1,7 +1,14 @@
 language: node_js
 node_js:
 - '8'
-script: npm test
+
+services:
+  - docker
+  - rabbitmq
+  - postgresql
+
+script: npm run test:ci
```

Comment:
Using the travis provided services is alright for now. But eventually we should move to running the tests within containers defined via our `docker-compose.yml`